lib/csharp-models-to-json/csharp-models-to-json.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Instead of just changing that and breaking who does not have 3.1, how about use multiple targets?
`<TargetFrameworks>netcoreapp2.0;netcoreapp3.1</TargetFrameworks>`

Reply:
I really wasn't aware that was possible. That's even better.